### ISSUE TITLE
Add ridge and lasso regression

### ISSUE THREAD

**Issue by fortierq** [mljar/mljar-supervised#446]
It seems to me that there is no regularized linear regression among the available algorithms, which are useful in my experience.  
I can try to add [Ridge](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html) and/or [Lasso](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Lasso.html), if you think it is worth it?

**Comment by pplonski** [comment]
@fortierq good idea, they can be added and left as a custom choice (can be selected in `algorithms` parameter in `AutoML()` constructor). Both algorithms can be added as they are from `scikit-learn` which is already in use.

If you are interested in adding both algorithms that would be fantastic! They should be add in similar way as `Linear` algorithm ([code](https://github.com/mljar/mljar-supervised/blob/master/supervised/algorithms/linear.py)), example how to set parameters for tuning (Random Forest [code](https://github.com/mljar/mljar-supervised/blob/master/supervised/algorithms/random_forest.py)).
